app/presenters/instance_presenter.rb

```diff
@@ -24,6 +24,10 @@ def active_user_count
     Rails.cache.fetch('active_user_count') { Redis.current.pfcount(*(0..3).map { |i| "activity:logins:#{i.weeks.ago.utc.to_date.cweek}" }) }
   end
 
+  def active_user_count_month(months: 6)
```

Comment:
defaulting to 6 makes the call site confusing, I would prefer if this took no default value